airflow/gcp/hooks/kubernetes_engine.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         self._client = None
         self.location = location
 
-    def get_client(self) -> container_v1.ClusterManagerClient:
```

Comment:
Can you add backward compatibility and deprecation warning?

Reply:
Sure. I've added it. Please check is it correct.